tests/test_test_tools/test_test_cases.py

```diff
@@ -141,7 +141,7 @@ def test_agent_set(self):
         """Test agent test set from path."""
         value = True
         key_name = "agent.logging_config.disable_existing_loggers"
-        self.set_config(key_name, value)
+        self.set_config(key_name, value, aev=True)
```

Comment:
@angrybayblade any suggestions for a meaningful way to test the `--aev` would be appreciated

Reply:
I implemented `TestConfigCasesAEV` (see https://github.com/valory-xyz/open-aea/pull/381/commits/9c6e9a7501c3679bc92c6a8f62266a94e4623b56) that provides a meaningful way to test the config overwrite.

However, `--aev` should not exist, it should be default behaviour.